frontend/app/.server/domain/entities/address-validation.entity.ts

```diff
@@ -0,0 +1,11 @@
+export type AddressCorrectionResultEntity = Readonly<{
+  'wsaddr:CorrectionResults': {
```

Comment:
Set `'wsaddr:CorrectionResults` object with `Readonly<T>` as well... We might want to look a an external plugin to provide `DeepReadonly` decorator.

Reply:
[Fixed](https://github.com/DTS-STN/canadian-dental-care-plan/compare/9f72343f90ece2d71a6eeebd83b569b11a78b791..aae8be84d4cbacc68bb747414c3de210aabe3968)